docs/.vuepress/components/ListPackages.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div>
+    <div v-if="loading">
+      <p>Loading…</p>
+    </div>
+
+    <p v-if="rejected">
+      Unable to get the packages list.
+    </p>
+
+    <ul v-if="success">
+      <li
+        v-for="pkg in packages"
+        v-bind:key="pkg.package.name">
+        <a
+          v-bind:href="'https://github.com/ovh/manager/tree/master/' + pkg.package.repository.directory"
+          rel="noopener noreferrer"
+          target="_blank">
+          {{ pkg.package.name }}@{{ pkg.package.version }}
+        </a>
+        <br>
+        <small>
```

Comment:
`small` has a semantic meaning, it is used for small prints, this is not the case here

Reply:
It is! I don't want to display a the package description as the same font Size than its name.

Reply:
`fine prints` means `legal text`, not `small characters` 😅